frontend/app/forms/consent-and-waiver-of-hearing/consent-and-waiver-of-hearing.pdf.component.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import RenderPage from "../render-page.component.js";
+import PositionedString from "../pdf-rendering/positioned-string.component.js";
+import PositionedCheckmark from "../pdf-rendering/positioned-checkmark.component.js";
+import {
+  getJudicialDistrict,
+  getCounty
+} from "../form-common-options/form-common-options";
+
+export default function consentAndWaiverOfHearing_Pdf({ data, renderData }) {
```

Comment:
Just found this - react components should have capitalized names in their function names. I'll send a PR to fix.